.claude/skills/create-gatherer/SKILL.md

```
@@ -0,0 +1,423 @@
+  - **IP addresses** (node IPs, cluster IPs, load balancer IPs, endpoint IPs)
+  - **Hostnames and DNS names** (especially external-facing ones)
+  - **URLs** (API server URLs, webhook URLs, external service URLs)
+  - **Cloud provider identifiers** (AWS region, account ID, VPC ID; GCP project ID, region; Azure subscription, resource group, cloud name)
+  - **Infrastructure names** (cluster name in infrastructure status)
+  - **Certificate data** (PEM-encoded certificates, keys)
+  - **Secret references** (values, not names)
+  - **User identifiers** (usernames, email addresses, organization names)
+  - **MAC addresses**
+  - **Any field that could identify a specific customer or their infrastructure**
+
+  ### Sensitive data — fields that do NOT need anonymization:
```

markdownlint-cli2 (0.23.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)